gkeepapi/__init__.py

```diff
@@ -171,20 +168,20 @@ def refresh(self):
         self._auth_token = res["Auth"]
         return self._auth_token
 
-    def logout(self):
+    def logout(self) -> None:
         """Log out of the account."""
         self._master_token = None
         self._auth_token = None
         self._email = None
         self._device_id = None
 
 
-class API(object):
+class API:
     """Base API wrapper"""
 
     RETRY_CNT = 2
 
-    def __init__(self, base_url, auth=None):
+    def __init__(self, base_url: str, auth: APIAuth = None):
```

Comment:
Apparently the current kosher in type hinting is to explicitly use `Optional`  (or ` | None` in python 3.10) if any param defaults to `None`:

```diff
-    def __init__(self, base_url: str, auth: APIAuth = None):
+    def __init__(self, base_url: str, auth: Optional[APIAuth] = None):

```

https://adamj.eu/tech/2022/10/18/python-type-hints-implicit-optional-types/